src/mwocp68.ron

```diff
@@ -1,5 +1,13 @@
 (
-    all: [],
+    all: [
+        (0xe4, "MFR_PAGE_X", WriteByte, ReadByte),
+        (0xf0, "BOOT_LOADER_KEY", WriteBlock, Illegal),
+        (0xf1, "BOOT_LOADER_STATUS", Illegal, ReadBlock),
```

Comment:
No `BOOT_LOADER_WRITE` (which is also 0xf1)?

Reply:
We can't tolerate different commands with the same command code -- and nor should we. ;)  I'll add a comment explaining what's going on.